src/utils.py

```diff
@@ -0,0 +1,35 @@
+import logging 
+from pathlib import Path
+from typing import Tuple
+
+import numpy as np
+import pandas as pd
+
+from config import Config
+
+log = logging.getLogger(__name__)
+
+def normalize(x: np.ndarray) -> np.ndarray:
+
+    return (x - np.mean(x)) / np.std(x)
+
+def load_data() -> Tuple[np.ndarray, np.ndarray]:
+    
+    data_dir = Path(__file__).parent.parent / 'data/CAMELS'
+    
+    # TODO add the input and output vars to config file
+    runoff_pd = pd.read_csv(data_dir / 'runoff_mm.csv')
+    precip_pd = pd.read_csv(data_dir / 'precipitation_mm.csv')
+    # attributes_pd = pd.read_csv(data_dir / 'attributes.csv')
+
+    # Calculating the mean for each basin across years.
+    runoff_mean = runoff_pd.mean().values[1:]
+    precip_mean = precip_pd.mean().values[1:]
```

Comment:
Likely worth a quick comment as to why the first value isn't used.

Reply:
probably better to slice before the `mean()` aggregation then if the mean of the year isn't meaninful/needed.

Reply:
This was how the code was implemented in the Colab notebook, but I agree it's not the best practice to slice after an operation. We can change this later and add as a TODO